typings/single-spa.d.ts

```diff
@@ -91,6 +91,29 @@ declare module "single-spa" {
     customProps?: T | CustomPropsFn<T>;
   };
 
+  interface SingleSpaNewAppStatus {
```

Comment:
I wonder if there are situations where the new status could be something besides the four here? Thinking through it, I think these four statuses are probably the only ones, but am not 100% sure.

Reply:
Hi @joeldenning 
I would be happy to add tests for these new types.   I apologize for my ignorance though; I don't think I understand how.

- I have read through https://www.npmjs.com/package/tsd    ("These .test-d.ts files will not be executed, and not even compiled in the standard way." )
- I see that `lib.dom.d.ts` has the type definition for window.addEventListener()
 `addEventListener(type: string, listener: EventListenerOrEventListenerObject, options?: boolean | AddEventListenerOptions): void;`


But all I can understand to do would be to add this to the test file:
```
window.addEventListener("single-spa:routing-event", ((evt: CustomEvent<SingleSpaCustomEventDetail>) => {
  expectType<SingleSpaCustomEventDetail>(evt.detail);
}) as EventListener);
```

But I'm not sure I'm actually testing anything there, right?

Reply:
Yes that code is the correct code to put into the test. It will ensure that typescript can compile it.

You can also try to do things like `expectType<string>(evt.detail.oldUrl)` to assert individual properties on the detail.

Reply:
Ok, thank you.   I pushed a change with a type test.